cve_bin_tool/checkers/curl.py

```diff
@@ -27,7 +27,7 @@ def get_version(lines, filename):
     """
     regex = [r"curl ([678]+\.[0-9]+\.[0-9]+)"]
     version_info = dict()
-    if filename[::-1].startswith(("curl")[::-1]):
+    if filename.find(("curl")) >= 0:
```

Comment:
Thank you for doing this update!  Could you replace the double brackets with single ones, making this

`if filename.find("curl") >= 0:`

I'm actually sort of surprised that Black doesn't have a rule to do that for you, but it didn't get caught last time either.

Reply:
I have resolved the code as you said. Please check the PR again.